microsoft-edge/extensions-chromium/getting-started/manifest-format.md

```diff
@@ -196,6 +197,25 @@ For reference information about each field, see [Manifest file format (V2)](http
 }
 ```
 
+---
+
+
+<!-- ====================================================================== -->
+## Changes in manifest.json from V2 to V3
+
+| V3 | V2 |
+|---|---|
+| `  "manifest_version": 3,` | `"manifest_version": 2,` |
+| removed | `"action": ...,` |
+| `"action": {...},` | `"browser_action": {...},` |
+| `"action": {...},` | `"page_action": {...},` |
+| removed | `"persistent": false,` |
+| If `background` is included, `service_ worker` is required | If `background` is included, `service_ worker` is optional |
+| `"host_permissions": [...],` | `"host_permissions": ...,` |
+| `"version_name": "aString",` | `"version_name": ...,` |
```

Comment:
"version_name" has not changed from MV2 to MV3. So, this line needs to be removed.

Reply:
assessment: On that line, V3 listing happens to use ... without quotes wrapping it.  V2 listing happens to use placeholder text wrapped in quote.  The inconsistent conventions gave a false diff.  Quick fix is to remove the table row, which I'm doing.